config/default.yml

```diff
@@ -2202,6 +2202,7 @@ Style/AccessModifierDeclarations:
   SupportedStyles:
     - inline
     - group
+  AllowModifiersOnSymbols: true
```

Comment:
You'll have to update `VersionChanged` as well.

Reply:
Added version `0.81`